cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -355,11 +450,23 @@ func getNodeNameForPVC(srcClient client.Client, namespace string, pvcName string
 	return "", nil
 }
 
-func garbageCollect(srcClient client.Client, destClient client.Client, labels map[string]string, endpoint string, namespace mappedNameVar) error {
+// getRsyncPassword returns a random password for rsync
+func getRsyncPassword() string {
+	var letters = []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	random.Seed(time.Now().UnixNano())
+	password := make([]byte, 6)
+	for i := range password {
+		password[i] = letters[random.Intn(len(letters))]
+	}
+	return string(password)
+}
+
+func garbageCollect(srcClient client.Client, destClient client.Client, labels map[string]string, endpoint endpointType, namespace mappedNameVar) error {
 	srcGVK := []client.Object{
 		&corev1.Pod{},
 		&corev1.ConfigMap{},
 		&corev1.Secret{},
+		&corev1.ServiceAccount{},
```

Comment:
is this list pending some more resources for garbage collection? I could be wrong but I recall more RBAC resources that were added in the lib needing garbage collection